Dockerfile

```diff
@@ -1,27 +1,29 @@
-FROM node:lts-alpine3.11
+FROM node:lts-alpine3.11 AS base
 
 # Versions
 # ARG NODE_VERSION=12.14.0 # Alpine 3.11 tracks NodeJS 12.x LTS
 # ARG NPM_VERSION=6.13.4 # Alpine 3.11 tracks NPM 6.x
-ARG SAILS_VERISON=^1.0.0
-
-# Variables
-ENV WORKDIR "/cppd-medical-report-sails"
 
-# Copy in files
-WORKDIR ${WORKDIR}
+ARG SAILS_VERISON=^1.0.0
+WORKDIR /app
 
-# Install SailsJS
-RUN npm install sails@${SAILS_VERISON} -g
+# Need to install python to compile bcrypt dependency
+RUN apk --no-cache add --virtual native-deps \
+  g++ gcc libgcc libstdc++ linux-headers autoconf automake make nasm python git && \
+  npm install --quiet node-gyp -g
 
-# Copy in build files
-COPY package.json package-lock.json ./
+# Package files
+COPY . /app/
 
 # Install dependencies
 RUN npm install
 
-# Copy in the rest of the files
-COPY . ./
+FROM node:lts-alpine3.11 AS final
+WORKDIR /app
+COPY --from=base /app /app
+
+# Install SailsJS
+RUN npm install sails@${SAILS_VERISON} -g
```

Comment:
Could we run this before the files are copied from the last container I'm pretty sure this will speed up the container build (admit I could be wrong here).

Reply:
moved it up